cmd/cloud-controller-manager/app/options/options.go

```diff
@@ -44,7 +44,7 @@ import (
 	"k8s.io/kubernetes/pkg/controller"
 
 	// add the kubernetes feature gates
-	_ "k8s.io/kubernetes/pkg/features"
+	_ "k8s.io/kubernetes/cmd/controller-manager/pkg/features"
```

Comment:
in `NewCloudControllerManagerOptions` add `runtime.Must(features.AddCurrentKubernetesSpecificFeatureGates(utilfeature.DefaultMutableFeatureGate))` and this does what you want, right?

Reply:
Moved the call from register here. Thanks

Reply:
Switched back to using init block because calling it here would fail the integration test by close feature gates early which blocked future features to be added.